Develop - patches #249, #83

smacp opened the pull request:
Could you please consider merging this branch. It contains patches for issues #249, #83 and options to omit the file node date and JMS file references from extracted xlf.

pimolo commented:
:+1:

Nyholm commented:
Thank you for this PR. I see that you done massive work. 
But to make it easier for us to review and discuss the changes I would prefer that you have one fix per pull request.

smacp commented:
Hi

Yes I made the fixes for #249 and #83 in the 'develop' branch in my fork.

This was a matter of urgency for me as one of my apps is using
JMSTranslationBundle with 25 languages and 12,000 words and these issues
were preventing successful extracts. The branch also moved on when I added
fixes to:

1. Validate PHPParser_Node_Scalar_String instances in the form extractor
2. Added additional config options to optionally add the xlf file date and
JMS file refs
3. PHP unit test fixes
4. Preserve custom trans unit attributes (I add machine translation dates
to my trans units)

As everyone always is I am a little pushed for time at the moment. Is it
possible to review the merge of the 'develop' branch without breaking it
down into individual PRs?

Thanks

On 23 March 2016 at 06:59, Tobias Nyholm <notifications@github.com> wrote:

> Thank you for this PR. I see that you done massive work.
> But to make it easier for us to review and discuss the changes I would
> prefer that you have one fix per pull request.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/schmittjoh/JMSTranslationBundle/pull/253#issuecomment-200213829>
>



-- 

Stuart MacPherson
Senior Developer
Purple
- - - - - - - - - - - - - - - - - - -
t : +44 (0) 333 101 4488 <+443331014488>
l : link with me <https://www.linkedin.com/profile/view?id=240270313>

w : purple.ai
f : like purple <https://www.facebook.com/PurpleWiFi>
t : follow purple <https://twitter.com/PurpleWiFi>
i : learn about purple <https://www.youtube.com/watch?v=VATT9rNw9Vk>
- - - - - - - - - - - - - - - - - - -
d : read email disclaimer <http://www.purple.ai/email-disclaimer/>

Nyholm commented:
Fixed in 1.3.0
